lib/bindings/bindings.cc

```diff
@@ -207,12 +205,11 @@ BindingInstance::bind_value(const char *name, int value)
     lua_getglobal(this->lua, start);
     isnil = lua_isnil(this->lua, -1);
     lua_pop(this->lua, 1);
+    Debug("lua", "isnil? %s", isnil ? "yes" : "no");
 
-    if (isnil) {
-      lua_pushvalue(this->lua, value);
-      lua_setglobal(this->lua, start);
-      bound = true;
-    }
+    lua_pushvalue(this->lua, value);
+    lua_setglobal(this->lua, start);
+    bound = true;
```

Comment:
Now that we always set or update the value, can `bound` ever be false? If not, remove it.